.travis.yml

```diff
@@ -0,0 +1,32 @@
+os: osx
+osx_image: xcode10.2
+compiler:
+  - clang
+  - gcc
+
+addons:
+  homebrew:
+    packages:
+    - cmake
+    - conan
+    - nasm
+    - llvm@6
+
+jobs:
+  include:
+    - script:
+        - export CC=/usr/local/opt/llvm\@6/bin/clang
```

Comment:
We shouldn't need to export CC and CXX when using a conan profile. Have you tried without this?

Reply:
I will re-check that, there was some confusion with the compiler path ... but it's probably fixed now.
Will write back after verification.